content/blog/an-example-application-for-processing-stock-market-trade-data-on-the-mapr-converged-data-platform.md

```diff
@@ -0,0 +1,291 @@
+
+<img src="https://github.com/mapr-demos/finserv-application-blueprint/raw/master/images/zepdash.png" width=400px>
+
+Here's what you need to do to set up a dashboard like that:
+
+#### Install and Configure Zeppelin
+
+We're going to assume you've already installed Apache Zeppelin and configured its Spark SQL interpreter to access Hive tables. If you haven't done that, then follow the instructions for installing Zeppelin in this blog post:
+
+https://community.mapr.com/docs/DOC-2029-how-to-use-spark-pyspark-with-zeppelin-on-mapr-cdp-draft
+
+#### Create Hive tables
+
+Import the bank company names into a new hive table. Later, we'll map these names to bank IDs with a table join in Zeppelin. 
+
+```
+hive
+  hive> CREATE TABLE banks(id int, name string) ROW FORMAT DELIMITED FIELDS TERMINATED BY ',';
+  hive> LOAD DATA LOCAL INPATH '/home/mapr/finserv-application-blueprint/resources/bank_list.csv' INTO TABLE banks;
+```
+
+Run this command to load streaming data into a Hive table:
+
+```
+/opt/mapr/spark/spark-2.0.1/bin/spark-submit --class com.mapr.demo.finserv.SparkStreamingToHive /home/mapr/nyse-taq-streaming-1.0.jar --topics /user/mapr/taq:trades --table streaming_ticks
+```
+
+That command will only see *new* messages in the trades topic because it tails the log, so when it says "Waiting for messages" then run the following command to put more trade data into the stream:
+
+```
+java -cp `mapr classpath`:/home/mapr/nyse-taq-streaming-1.0.jar com.mapr.demo.finserv.Run producer /home/mapr/finserv-application-blueprint/data/ /user/mapr/taq:trades
+```
+
+#### Create a Zeppelin notebook
+
+Open Zeppelin in a web browser.
+
+Zeppelin divides notebooks into subsections called *paragraphs*. Create a new notebook in Zeppelin, then in a new paragraph enter the following and press the 'Play' icon (or keyboard shortcut, Shift+Enter):
+
+```
+%sql show tables
+```
+
+You should see the `banks` and `streaming_ticks` tables in the list. If not, look for an error in the logs under `/opt/zeppelin/logs/`.
+
+We've provided a sample Zeppelin notebook which includes some sample SQL queries and charts to get you started. Load the `finserv-application-blueprint/resources/sample_zeppelin_notebook.json` file with the Import feature in the Zeppelin web UI. After you've imported it, you should see a new notebook called "Stock Exchange Analysis" in Zeppelin, which looks like this:
+
+<img src = "https://github.com/mapr-demos/finserv-application-blueprint/raw/master/images/zepdash2.png" width=600px>
+
+## (Optional) Generate More Data
+
+You can download 500MB more trade data with the following commands. Note, that these data files are kept in a separate GitHub repository in order to keep this one to a manageable size.      
```

Comment:
When you use "that", you don't need the comma after "Note". So it would be "Note that these..."